dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -369,6 +369,49 @@ def diff(self, other_profile: NumericStatsMixin, options: Dict = None) -> Dict:
                 "and '{}'".format(cls.__name__, other_profile.__class__.__name__)
             )
 
+        len_self_bin_counts = 0
+        self_bin_counts = self._stored_histogram["histogram"]["bin_counts"]
+
+        try:
+            if self_bin_counts.all() is not None and len(self_bin_counts) > 0:
+                len_self_bin_counts = len(self_bin_counts)
+
+            # re-calculate `self` histogram
+            if not len_self_bin_counts == 10 and self_bin_counts.all() is not None:
```

Comment:
do we want to prepare this for allowing `20` to be a value as well?
making it a property pulled from options with a default of 10?

Reply:
I'm definitely game to add! My though is to keep this PR small and focused on just adding the functionality when `self.diff()` is called. Then have a follow-up for configuring `PSI` through options. Thoughts?

Reply:
I suggest to keep it clean, i would at the very least set this as a variable at the top to ensure consistency of all changes in the future.

e.g. just change `10` to `num_psi_bins`
then we can always do a `num_psi_bins = options.get(...., 10)` when we are ready

Reply:
Otherwise, if someone accidentally changes one of these, these calcs will be off entirely

Reply:
Fair -- I'll add that

Reply:
added